src/stores/data.ts

```diff
@@ -501,12 +529,83 @@ const useDataStore = create<DataStore>()(
         reader.readAsText(file);
       }),
 
-    hasMultiColumnMeasures: () => {
-      const { columns } = get();
-      return Object.values(columns).some(
-        (column) =>
-          column.standardizedVariable?.identifier === get().getAssessmentToolConfig().identifier
+    hasMultiColumnMeasures: () =>
+      get().getMappedMultiColumnMeasureStandardizedVariables().length > 0,
+
+    loadConfigOptions: async () => {
+      try {
+        const dirNames = await getConfigDirs();
+        set({ configOptions: dirNames });
+      } catch (error) {
+        // TODO: show a notif error
+        set({ configOptions: [] });
+      }
+    },
+
+    loadConfig: async (configName: string) => {
+      try {
+        const config = await import(`../../configs/${configName}/config.json`);
+        set({ config: config.standardizedVariables });
+      } catch (error) {
+        // TODO: show a notif error
+      }
+    },
+
+    setSelectedConfig: (configName: string | null) => {
+      if (configName) {
+        set({ selectedConfig: configName });
+        get().loadConfig(configName);
+      } else {
+        set({ selectedConfig: 'Neurobagel' });
+        get().loadConfig('Neurobagel');
+      }
+    },
+
+    getTermOptions: async (standardizedVariable: StandardizedVariable) => {
```

Comment:
Not sure running an API fetcher inside the store is the best idea here. Why not handle this logic in a custom hook, call it from the App, e.g. when the whole App loads or even when the user selects a config, and then just handle storing the terms in the store?